README.rst

```diff
@@ -1,69 +1,105 @@
++------------+-----------------------------------------------------------+
+
+.. contents:: Table of Contents
+   :depth: 1
+   :backlinks: none
 
 About
 =====
 
 The **nidaqmx** package contains an API (Application Programming Interface)
-for interacting with the NI-DAQmx driver. The package is implemented in Python.
-The package is implemented as a complex,
-highly object-oriented wrapper around the NI-DAQmx C API using the
+for interacting with the NI-DAQmx driver.
+
+Documentation
+-------------
+
+You can find the latest API documentation for the **nidaqmx** package on
+`Read the Docs <http://nidaqmx-python.readthedocs.io/en/latest>`_.
+
+Refer to the `NI-DAQmx Help <http://digital.ni.com/express.nsf/bycode/exagg4>`_
+for API-agnostic information about NI-DAQmx or measurement concepts. NI-DAQmx
+Help also installs locally with the full version of NI-DAQmx.
+
+Implementation
+--------------
+
+The package is implemented in Python as a complex, highly object-oriented
+wrapper around the NI-DAQmx C API using the
 `ctypes <https://docs.python.org/2/library/ctypes.html>`_ Python library.
 
+Supported NI-DAQmx Driver Versions
+----------------------------------
+
 **nidaqmx** supports all versions of the NI-DAQmx driver that ships with the C
-API. The C API is included in any version of the driver that supports it. The
-**nidaqmx** package does not require installation of the C header files.
+API. The **nidaqmx** package does not require installation of the C header
+files.
 
 Some functions in the **nidaqmx** package may be unavailable with earlier
-versions of the NI-DAQmx driver. Visit the
-`ni.com/downloads <http://www.ni.com/downloads/>`_ to upgrade your version of
-NI-DAQmx.
+versions of the NI-DAQmx driver. Refer to the Installation section for details
+on how to install the latest version of the NI-DAQmx driver.
+
+Operating System Support
+------------------------
 
 **nidaqmx** supports Windows and Linux operating systems where the NI-DAQmx
 driver is supported. Refer to
 `NI Hardware and Operating System Compatibility <https://www.ni.com/r/hw-support>`_
 for which versions of the driver support your hardware on a given operating
 system.
 
+Python Version Support
+----------------------
+
 **nidaqmx** supports CPython 3.8+ and PyPy3.
 
 Installation
 ============
 
-Running **nidaqmx** requires NI-DAQmx to be installed. Visit
-`ni.com/downloads <http://www.ni.com/downloads/>`_ to download the latest
-version of NI-DAQmx. None of the recommended **Additional items** are required
-for **nidaqmx** to function, and they can be removed to minimize installation
-size. It is recommended you continue to install the **NI Certificates** package
-to allow your Operating System to trust NI built binaries, improving your
-software and hardware installation experience.
-
-**nidaqmx** can be installed with `pip <http://pypi.python.org/pypi/pip>`_::
+**nidaqmx** can be installed with `pip <http://pypi.python.org/pypi/pip>`_ from
```

Comment:
This sounds like pip is from PyPI, which it is, but that's not the point you're trying to make.

Also, PyPI should be bicapitalized.

How about "You can use pip to download nidaqmx from PyPI and install it:"?